src/services/chatgpt.ts

```diff
@@ -1,29 +1,23 @@
-import { OPENAI_API_KEY } from '../env'
-const BASE_URL = 'https://api.openai.com/v1'
+const BASE_URL = 'https://api.openai.com/v1';
```

Comment:
you could add this to a config file, so it's easier to change if the open ai changed URL or version without going through the code.